### ISSUE TITLE
Improper handling of table parameters in event handlers

### ISSUE THREAD

**Issue by robot256** [jodli/CreativeMod#13]
In 0.17.79, using Creative Mod version 1.2.0, I received an error message when the console print instructions for "on_player_pipette" and "on_built_entity" are encountered.  Looks like they should be serialized (e.g. by __OpteraLib__.logger.tostring) or ignored.  Representative error message from the attached log file:

```
3259.628 Error MainLoop.cpp:1195: Exception at tick 3198415: The mod Creative Mod caused a non-recoverable error.
Please report this error to the mod author.

Error while running event creative-mod::on_player_pipette (ID 92)
__creative-mod__/scripts/events.lua:642: attempt to concatenate local 'param' (a table value)
stack traceback:
	__creative-mod__/scripts/events.lua:642: in function <__creative-mod__/scripts/events.lua:641>
	(...tail calls...)
	__creative-mod__/scripts/events.lua:1104: in function <__creative-mod__/scripts/events.lua:1078>
```

[factorio-current.log](https://github.com/jodli/CreativeMod/files/4014128/factorio-current.log)

**Comment by jodli** [comment]
Hi!
I will be looking at this today!

Thanks for reporting!

**Comment by jodli** [comment]
I'm really sorry, but I can't seem to be able to reproduce this exception.
What item did you use the pipette on and what item did you try to build when this error happened?

**Comment by robot256** [comment]
At the time, I believe I was doing it with train locomotives and wagons.  It looks like this has already been fixed as of Factorio 1.0.  It displays a properly serialized list of attributes for the built entity or pipetted item.  Thanks!
